File: car-ui/src/app/shared/module/OrderSummary.ts
export class OrderSummary {
    orderId: string;
    message: string;
    orderStatus: string;

    constructor(orderId: string, message: string, orderStatus: string) {
        this.orderId = orderId;
        this.message = message;
        this.orderStatus = orderStatus;
    }
}

File: order-service/src/main/java/com/nashtech/order/restapi/response/OrderSummary.java
package com.nashtech.order.restapi.response;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

@Builder
@NoArgsConstructor
@AllArgsConstructor
@Getter
public class OrderSummary {
    private String orderId;
    private String orderStatus;
    private String message;

}